README.md

```diff
@@ -31,13 +31,31 @@ Pythoshop complements existing image processing libraries in the Python ecosyste
 ## Installation
 
 ```bash
-# you can't use this for now since the package hasn't be published yet.
+# you can't use this for now since the package hasn't been published yet.
 $ pip install pythoshop
 ```
 
 ## Usage
 
-Navigate to the folder that you want to download the pythoshop package.
+First, you need to make sure you have installed conda and poetry.
+
+**To install Anaconda:**
+
+1. Download the Anaconda Installer:
+- Visit the [Anaconda Download page](https://www.anaconda.com/download)
+- Choose the installer for your operating system (Windows, macOS, or Linux).
+- Select the version for Python 3.x.
+2. Run the Installer:
+- Windows: Open the downloaded .exe file and follow the instructions. It's recommended to check the option to "Add Anaconda to my PATH environment variable" for easy use in the command prompt, but be aware this can interfere with other software.
+- macOS or Linux: Open Terminal, navigate to the directory containing the downloaded script, and run it with bash Anaconda3-xxxxxx.sh, following the on-screen instructions.
+3. Verify Installation:
+Open your command line interface (CLI) and type `conda list`. If Anaconda is installed properly, you should see a list of installed packages.
+
+**To install `poetry`, follow the instructions below:**
+
+we recommend installing `Poetry` using their official installer while referring to their official [poetry documentation](https://python-poetry.org/docs/) for detailed installation instructions and support.
+
+Now navigate to the folder that you want to download the pythoshop package.
```

Comment:
how can they install?
they need to run poetry install from the root of the Pythoshop folder, can we add this?